src/Components.elm

```diff
@@ -167,33 +169,54 @@ player :
     , sendTextToKodiMsg : msg
     , scanVideoLibraryMsg : msg
     , scanMusicLibraryMsg : msg
+    , volumeSlider : Element msg
     }
     -> Element msg
-player { currentlyPlaying, playPauseMsg, skipMsg, reverseMsg, muteMsg, repeatMsg, shuffleMsg, controlMenu, controlMenuMsg, sendTextToKodiMsg, scanVideoLibraryMsg, scanMusicLibraryMsg } =
+player { currentlyPlaying, playPauseMsg, skipMsg, reverseMsg, muteMsg, repeatMsg, shuffleMsg, controlMenu, controlMenuMsg, sendTextToKodiMsg, scanVideoLibraryMsg, scanMusicLibraryMsg, volumeSlider } =
     row [ height (px 70), width fill, alignBottom ]
-        [ row [ height fill, width (fillPortion 1), Background.color (rgb 0.2 0.2 0.2), alignBottom, padding 10, spacing 30 ]
-            [ el [ centerX ] (reverseButton { reverseMsg = reverseMsg })
-            , el [ centerX ] (playButton { playPauseMsg = playPauseMsg })
-            , el [ centerX ] (skipButton { skipMsg = skipMsg })
+        [ playControlRow reverseMsg playPauseMsg skipMsg
+        , column [ width (px 70) ]
+            [ image [ alignLeft ]
+                { src = "https://via.placeholder.com/70"
+                , description = "Hero Image"
+                }
             ]
-        , column [ height fill, width (fillPortion 2) ]
-            [ row [ height (px 70), width fill, Background.color (rgb 0.2 0.2 0.2), alignBottom, padding 10, spacing 30 ]
-                [ image [ alignLeft ]
-                    { src = "https://via.placeholder.com/70"
-                    , description = "Hero Image"
-                    }
-                , el [ Font.color (Element.rgb 0.6 0.6 0.6), Font.size 18, Font.family [ Font.typeface "Open Sans", Font.sansSerif ] ] (text currentlyPlaying.title)
-                , el [ alignRight, Font.color (Element.rgb 0.6 0.6 0.6), Font.size 18, Font.family [ Font.typeface "Open Sans", Font.sansSerif ] ] (text (String.fromInt currentlyPlaying.duration))
-                ]
+        , currentlyPlayingColumn currentlyPlaying volumeSlider
```

Comment:
I'm passing `volumeSlider` for now. This needs to be changed.

Reply:
what are you thinking of changing it to?

Reply:
ah I see nvm